src/main/java/thaumicboots/item/boots/voidwalker/ItemElectricVoidwalkerBoots.java

```diff
@@ -146,84 +110,8 @@ public void onArmorTick(final World world, final EntityPlayer player, final Item
         }
     }
 
-    @SideOnly(Side.CLIENT)
-    private void particles(final World world, final EntityPlayer player) {
-        final FXWispEG fx = new FXWispEG(
-                world,
-                player.posX + (Math.random() - Math.random()) * 0.5D,
-                player.boundingBox.minY + 0.05D + (Math.random() - Math.random()) * 0.1D,
-                player.posZ + (Math.random() - Math.random()) * 0.5D,
-                player);
-        ParticleEngine.instance.addEffect(world, fx);
-    }
-
-    @Override
-    public ISpecialArmor.ArmorProperties getProperties(EntityLivingBase player, ItemStack armor, DamageSource source,
-            double damage, int slot) {
-        if (source.isUnblockable()) {
-            return new net.minecraftforge.common.ISpecialArmor.ArmorProperties(0, 0.0D, 0);
-        } else {
-            double absorptionRatio = getBaseAbsorptionRatio() * getDamageAbsorptionRatio();
-            int energyPerDamage = getEnergyPerDamage();
-            double damageLimit = energyPerDamage <= 0 ? 0
-                    : (25 * ElectricItem.manager.getCharge(armor)) / energyPerDamage;
-            return new net.minecraftforge.common.ISpecialArmor.ArmorProperties(0, absorptionRatio, (int) damageLimit);
-        }
-    }
-
-    @Override
-    public int getArmorDisplay(EntityPlayer player, ItemStack armor, int slot) {
-        if (ElectricItem.manager.getCharge(armor) >= getEnergyPerDamage()) {
-            return (int) Math.round(20D * getBaseAbsorptionRatio() * getDamageAbsorptionRatio());
-        } else {
-            return 0;
-        }
-    }
-
-    @Override
-    public void damageArmor(EntityLivingBase entity, ItemStack stack, DamageSource source, int damage, int slot) {
-        ElectricItem.manager.discharge(stack, damage * getEnergyPerDamage(), 0x7fffffff, true, false, false);
-    }
-
-    public double getDamageAbsorptionRatio() {
-        return 2.25D;
-    }
-
-    private double getBaseAbsorptionRatio() {
-        return 0.15D;
-    }
-
-    public int getEnergyPerDamage() {
-        return energyPerDamage;
-    }
-
-    @Override
-    public boolean canProvideEnergy(ItemStack itemStack) {
-        return false;
-    }
-
-    @Override
-    public double getMaxCharge(ItemStack itemStack) {
-        return maxCharge;
-    }
-
-    @Override
-    public int getTier(ItemStack itemStack) {
-        return 2;
-    }
-
-    @Override
-    public double getTransferLimit(ItemStack itemStack) {
-        return transferLimit;
-    }
-
     @Override
-    public Item getChargedItem(ItemStack itemStack) {
-        return this;
-    }
-
-    @Override
-    public Item getEmptyItem(ItemStack itemStack) {
-        return this;
+    public EnumRarity getRarity(final ItemStack stack) {
```

Comment:
useless override

Reply:
That's not, because it needs to be purple, for some reason the rarity system doesn't work as variables, dunno why